content/millennium/r4/entities/location.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Location | R4 API
+---
+
+# Location
+
+* TOC
+{:toc}
+
+## Overview
+
+The Location resource describes physical places where healthcare services are provided.  In Millennium, facilities are the top level of the patient location hierarchy.  A facility is also an Organization resource at which patient locations are associated.  All facilities are organizations, but not all organizations are facilities.  A location can also be an ambulatory patient care area like a clinic or an emergency room.  The location hierarchy from highest to lowest is facility, building, nursing unit, room, and bed.
+
+
+The following fields are returned if valued:
+
+* [Location id](http://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](http://hl7.org/fhir/R4/location-definitions.html#Location.status){:target="_blank"}
+* [Name](http://hl7.org/fhir/R4/location-definitions.html#Location.name){:target="_blank"}
+* [Alias](http://hl7.org/fhir/R4/location-definitions.html#Location.alias){:target="_blank"}
+* [Mode](http://hl7.org/fhir/R4/location-definitions.html#Location.mode){:target="_blank"}
+* [Telecom](http://hl7.org/fhir/R4/location-definitions.html#Location.telecom){:target="_blank"}
+* [Address](http://hl7.org/fhir/R4/location-definitions.html#Location.address){:target="_blank"}
+* [Physical Type](http://hl7.org/fhir/R4/location-definitions.html#Location.physicalType){:target="_blank"}
+* [Managing Organization](http://hl7.org/fhir/R4/location-definitions.html#Location.managingOrganization){:target="_blank"}
+* [Part Of](http://hl7.org/fhir/R4/location-definitions.html#Location.partOf){:target="_blank"}
```

Comment:
Part of-> partOf

@praneethk2094 I am getting this correct? The field listed here should be in the same case they are on spec?

Reply:
This is done as per this Jira:
https://jira2.cerner.com/browse/OPENSVC-15097